<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>255</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clean">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>255</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_export">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>305</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Экспортировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_run">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>305</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>345</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Если максимальный элемент в таблице расположен после минимального, то поменять значения элементов
первой и второй строки между собой</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_array">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>46</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
